util/dvsim/FlowCfg.py

```diff
@@ -139,10 +138,22 @@ def __init__(self, flow_cfg_file, hjson_data, args, mk_config):
         # _expand and add the code at the start.
         self._expand()
 
-        # Construct the result_path Path variable after variable expansion.
-        self.results_path = (Path(self.scratch_base_path) / "reports" /
-                             self.rel_path / "latest")
-
+        # Construct the path variables after variable expansion.
+        self.results_local_dir = (Path(self.scratch_base_path) / "reports" /
+                                  self.rel_path / "latest")
+        self.results_local_page = (self.results_local_dir /
+                                   self.results_html_name)
+
+        self.results_server_path = (self.results_server_prefix +
+                                    self.results_server + "/" +
+                                    self.rel_path)
+        self.results_server_dir = self.results_server_path + "/latest"
```

Comment:
Note that this creates the same directory structure locally and for the reports server.